src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionTest.java

```diff
@@ -129,9 +131,11 @@ void subscribeExclusiveTopicAndPattern(VertxTestContext context) throws Throwabl
 
     @Test
     void subscriptionConsumerDoesNotExistBecauseNotCreated(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
-        String topic = "subscriptionConsumerDoesNotExistBecauseNotCreated";
-        kafkaCluster.createTopic(topic, 1, 1);
         String name = "my-kafka-consumer-does-not-exists-because-not-created";
+        String topic = "subscriptionConsumerDoesNotExist";
```

Comment:
why changing the name?

Reply:
Honestly, I do not remember but it is not so important i would say.

Reply:
So we can revert back to having the name as the method? I think it's the rule for other tests.

Reply:
done